src/main/java/com/itkonboarding/airport_gate/entities/Flight.java

```diff
@@ -0,0 +1,59 @@
+package com.itkonboarding.airport_gate.entities;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import javax.persistence.*;
+
+@Entity
+@JsonIgnoreProperties({"handler", "hibernateLazyInitializer"})
```

Comment:
Please remove this annotation. It's not necessary since we won't convert entities directly to josn.